DecipherNow/gm-fabric-dashboard
# 524 fix inspector search

This PR brings the approach I came up in searching for services to searching metrics.

To test this, go to Explorer for a microservice

The following should work:
- Enter a search term. You should be able to type naturally, and at a natural pause in typing, the search word or phrase should be appended to the URL as a search query.
- Enter a second search term. You should see things update and it should replace the existing search query
- Select a metric to open a chart. you should see a second key-value pair get added to the query string in the URL
- Do a hard refresh of the browser, the app should load exactly where you left off
- Click the back and forward buttons. The state should update to reflect what you've done

Resolves #524
